rules/0015-wazuh_rules.xml

```diff
@@ -315,7 +481,7 @@
   </rule>
 
   <rule id="598" level="5">
-    <category>ossec</category>
+    <category>wazuh</category>
     <if_sid>554</if_sid>
     <hostname>syscheck-registry</hostname>
     <group>syscheck,pci_dss_11.5,gpg13_4.13,gdpr_II_5.1.f,hipaa_164.312.c.1,hipaa_164.312.c.2,nist_800_53_SI.7,tsc_PI1.4,tsc_PI1.5,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
```

Comment:
We could maybe replace the installation path in the example (line 493): `/var/ossec/active-response/bin/firewall-drop.sh delete` for something like: /INSTALLATION_PATH/active-response/bin/firewall-drop.sh delete

Reply:
Fixed